js/client/app/todoList/filters/applyPriorityFilter.js

```diff
@@ -0,0 +1,18 @@
+import { loadTodoList } from "../../../../serverAPI/user/loadTodoList.js";
+import { drawTodo } from "../../drawTodo.js";
+import { updateCountText } from "../updateCountText.js";
+
+export async function applyPriorityFilter(userID) {
+  const teamID = localStorage.getItem("teamID");
+  const userUL = document.getElementById(userID).querySelector("ul.todo-list");
+  const todoList = await loadTodoList(teamID, userID);
+
+  if (todoList.todoList === undefined) return;
+  userUL.innerHTML = "";
+
+  for (const todo of todoList.todoList) {
+    drawTodo(userUL, todo, true);
+  }
```

Comment:
```suggestion
  if (todoList.todoList === undefined) {
    return;
  }  
  userUL.innerHTML = "";

  for (const todo of todoList.todoList) {
    drawTodo(userUL, todo, true);
  }
```

이거는 취향차이일수도 있는데 한 스코프 안에서는 하나의 형식으로 통일해서 사용해주시면 더 보기 좋을거 같습니다!